files/keyvalue

```diff
@@ -0,0 +1,12 @@
+# Grok patterns for Key-Value pair messages, where the useful payload is
+# encoded as a list of key=value strings.
+#
+# All these should put the useful payload into the variable KEYVALUEDATA,
+# to ensure it will be parsed properly by the kv filter
+# - JUBE suite messages
+
+
+JUBE_MSG %{DATA} JUBE %{GREEDYDATA:KEYVALUEDATA}
+
+
+KEYVALUE_MSG (%{JUBE_MSG})
```

Comment:
you don't use the group reference other then to construct an `or`-ed list of patterns, better make it a  non-captured group (that's what i meant instead of `unnamed`)